src/benchq/resource_estimation/v2/master.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+from .transformers import synthesize_clifford_t
+
+
+def run_resource_estimation_pipeline(
+    program_or_circuit,
+    error_budget,
+    estimator,
+    transformer=synthesize_clifford_t,
+):
+    transformed = transformer(program_or_circuit, error_budget)
```

Comment:
Not sure what a `transformer` is, but @mstechly says it needs a rework anyways so we won't worry about that right now.